CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## Develop - 2024-10-11
+### Added
+- Add new **ElectricStorage**, **ColdThermalStorage**, and **HotThermalStorage** parameters **max_duration_hours** and **min_duration_hours** to bound the energy duration of battery storage
```

Comment:
@atpham88 just noting that I updated this to add thermal storage to the description